vault/identity_store.go

```diff
@@ -455,6 +473,7 @@ func (i *IdentityStore) CreateOrFetchEntity(ctx context.Context, alias *logical.
 	newAlias := &identity.Alias{
 		CanonicalID:   entity.ID,
 		Name:          alias.Name,
```

Comment:
Can we sanitize the name here?

Reply:
Nope. We don't need to sanitize here. It is done in the utility that inserts item into memdb.